src/panels/config/areas/ha-config-area-page.ts

```diff
@@ -258,7 +258,13 @@ class HaConfigAreaPage extends SubscribeMixin(LitElement) {
           <div class="column">
             ${area.picture
               ? html`<div class="img-container">
-                  <img src=${area.picture} /><ha-icon-button
+                  <img
+                    alt=${this.hass.localize(
+                      "ui.panel.config.areas.area_picture"
```

Comment:
New string.

Reply:
I think we should get rid of the string and the alt text should just be the name of the area.  In a future PR, we can let the user create/edit the text in the picture upload dialog.